pyproject.toml

```diff
@@ -115,6 +115,7 @@ xlwt = "1.2.0"
 "zope.deprecation" = "4.4.0"
 "zope.interface" = "4.6.0"
 "zope.sqlalchemy" = "^1.2"
+sentry-sdk = "^0.16.2"
```

Comment:
Uparrow does little in version 0, only changes patch version. Maybe want `">=0.16.2"` or `">=0.16.2,<1"`. The latter is what you probably thought you meant. But really if we just want their highest version, probably we shouldn't even do `"<1"`.